Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Calibration/CompositorWrapper.cs

```diff
@@ -1,47 +1,66 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-// Enable this preprocessor directive in your player settings as needed.
-#if COMPOSITOR_PLUGIN_AVAILABLE
-using SpectatorView;
-#endif
-
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
 {
-    internal class CompositorWrapper
+    internal class CompositorWrapper : Singleton<CompositorWrapper>
     {
-        public static RenderTexture GetDSLRFeed()
+        /// <summary>
+        /// CompositionManager used to obtain a DSLR stream
+        /// </summary>
+        [Tooltip("CompositionManager used to obtain the DSLR stream")]
+        [SerializeField]
+        private CompositionManager compositionManager = null;
+
+        public RenderTexture GetDSLRFeed()
```

Comment:
>GetDSLRFeed [](start = 29, length = 11)

I think I'd been trying to call this the "VideoCamera" elsewhere

Reply:
Sounds good to me. I'll change this now.